### ISSUE TITLE
BloombergLP::bdldfp::Decimal128 is ignoring std::fixed manipulator when output to stream

### ISSUE THREAD

**Issue by skwllsp** [bloomberg/bde#244]
On MacOS (the demo program compiled with clang++) and Ubuntu (the demo program compiled with g++) there is a difference between output for double and for Decimal128 when std::fixed is used. Decimal128 do not handle propely the std::fixed manipulator. This is the demo program:
```
#include <bdl/bdldfp_decimal.h>
#include <bdl/bdldfp_decimalutil.h>
#include <iostream>
#include <iomanip>
#include <limits>

int main(int argc, char *argv[]) {

	double d = 0;
	std::cout << std::fixed << std::setprecision(6) << d << std::endl;

    BloombergLP::bdldfp::Decimal128 dec{0};
	std::cout << std::fixed << std::setprecision(6) << dec << std::endl;

	d = 1.1;
	dec = BloombergLP::bdldfp::Decimal128(1.1);
	std::cout << std::fixed << std::setprecision(6) << d << std::endl;
	std::cout << std::fixed << std::setprecision(6) << dec << std::endl;
}

And this is its output:

```
$ ./a.out                                                                                                                                       
0.000000
0
1.100000
1.100000000000000088817841970012523
```
```

**Comment by osubboo** [comment]
Ack.
Have to check, but there were significant revamp of the decimal support recently. Can be already fixed in the upstream.

**Comment by osubboo** [comment]
Yes, this code does not support precision manipulators for decimal types. The new versions in the upstream will correctly handle them. Expected in 3.6.0.

**Comment by hyrosen** [comment]
Aside from this being fixed in 3.6.0, the usage exhibited here is itself bad.  *DON'T DO THAT!*

Creating a decimal from a double in the way shown, by constructing the decimal from the double, is only appropriate if you do not know the provenance of the double, and believe that it may have come about as the result of an arbitrary calculation.

If you believe that the double value instead was the result of a conversion from a decimal value (which is certainly the case here, that being the "1.1" constant in the code), use bdldfp::DecimalConvertUtil::decimal128FromDouble(d) if you have the value in a variable, and 
BDLDFP_DECIMAL_DL(1.1) if you have the value as a manifest constant.  (With C++11 and BDE 3.6.0, you will also be able to say 1.1_d128, with our new user-defined literal support.)
